packages/common/src/authorization/projectMemberAbility.ts

```diff
@@ -56,12 +87,31 @@ export const projectMemberAbilities: Record<
         projectMemberAbilities.interactive_viewer(member, { can });
         can('manage', 'Dashboard', {
             projectUuid: member.projectUuid,
+            isPrivate: false,
+        });
+        // should not check space access when creating a space
+        can('create', 'Space', {
+            projectUuid: member.projectUuid,
         });
         can('manage', 'Space', {
             projectUuid: member.projectUuid,
+            isPrivate: false,
+        });
+        can('manage', 'SavedChart', {
+            projectUuid: member.projectUuid,
+            isPrivate: false,
+        });
+        can('manage', 'Dashboard', {
+            projectUuid: member.projectUuid,
+            access: {
+                $elemMatch: { userUuid: member.userUuid },
```

Comment:
They can only create/update/delete if their space role is Editor.
```suggestion
                $elemMatch: { userUuid: member.userUuid },
```